cmd/juju/controller/register.go

```diff
@@ -403,36 +525,37 @@ func (c *registerCommand) promptNewPassword(stderr io.Writer, stdin io.Reader) (
 	return password, nil
 }
 
-const errControllerConflicts = `WARNING: You already have a controller registered with the name %q. Please choose a different name for the new controller.
-
-`
-
 func (c *registerCommand) promptControllerName(store jujuclient.ClientStore, suggestedName string, stderr io.Writer, stdin io.Reader) (string, error) {
 	_, err := store.ControllerByName(suggestedName)
```

Comment:
Please skip this if suggestedName == "". It would previously always be non-empty, but that's not the case now.